docs/blog/2022-08-12-observing-randomness.md

```diff
@@ -0,0 +1,140 @@
+
+There are other things that can go wrong during the ceremony. One of the nodes might not be updated to the appropriate version of drand. The node operator might restart the node in the middle of the ceremony, which then sets the node at the wrong state. There might be connectivity problems which do not allow a node to correctly participate. And so on and so forth. Up until now, the coordination between operators has been through off-band channels (such as email, or Slack), which resulted in postponed or cancelled ceremonies. The difficulties in coordinating a ceremony through such means surfaced the need to have full visibility into the drand network and the state of each of the nodes participating in it, in order to ensure that all nodes are in the correct state and that everything happens at the right time and in the right order.
+
+The ultimate target of the LoE with regard to ceremony operations is to be able to carry out “async ceremonies”. In this setup, a timeframe will be set up for all operators executing the appropriate commands without everyone having to be online at the same time. This will become more important as the drand network grows and more participants in different timezones join the league. A monitoring tool that provides an accurate view of node and network state, as well as the operations that are being carried out is clearly an absolute necessity.
+
+## Monitoring architecture requirements
+
+To properly monitor the drand network we have some requirements:
+
+**Allow any LoE member to create their own monitoring infrastructure**
+
+We did not want to mandate using a specific metrics aggregation platform (e.g. Prometheus or InfluxDB), instead allowing for each organization to select the one that best meets their needs or with which they are most familiar.
+
+**Ease of deployment**
+
+We did not want to burden operators with complex deployment strategies, so we decided to modify the drand binary itself instead of adding a new binary to export the metrics.
+
+**Minimize the need for opening network ports**
+
+Since drand nodes are run by multiple organizations on their own networks, any changes to the firewall configuration increases friction and reduces the probability of success. Ideally no network ports other than the one used for generating randomness should need to be opened. At the very least no more incoming ports should be needed.
+
+**Use industry-standard formats where possible**
+
+To decrease the ramp-up time for integrating into an organization’s monitoring system, we need to use an industry-standard format to publish metrics.
+
+**Ability to see the state of the whole network from any point in the network**
+
+To allow all LoE members to create their own dashboards and alerts, we need to enable each node to provide a full view of the whole network.
```

Comment:
```suggestion
To allow all LoE members to create their own dashboards and alerts, we need to allow each node to provide a full view into the whole network.
```

Reply:
view of the whole network sounds better imo

Reply:
but does the node _provide_ a full view or _receive_ a full view?

Reply:
you can get a full view from any node, so the node provides a full view (for which it receives data from each of the other nodes)

Reply:
I like `enable` rather than `allow`, since it's not a permissions problem but rather a functionality problem.